### PR TITLE
Mistake 

### PR DESCRIPTION
**Pull request by sarinalalria**

### CODE REVIEW

**Comment by sarinalalria** [comment]
I will fix the guest.html when the other pull request is fixed.

**Review by Sinclair198 — APPROVED** [review]
all good.

**Review by YiShianLiu — APPROVED** [review]

**Review by nscheiber — APPROVED** [review]

File: express-mongodb/signup-app.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+app.use(bodyParser.urlencoded({
+    extended:true
+}))
+
+const mongodb = require('mongodb')
+const MongoClient = mongodb.MongoClient
+
+// Connection URI
+const url = 'mongodb+srv://gamequiz:gameQuizTeam@gamequiz.9o3my64.mongodb.net/test';
```
Comment (zeshengchen) [inline express-mongodb/signup-app.js:18]: do not hard-coded username and password in the code. please use .env file and store them in this file.

**Review by zeshengchen — CHANGES_REQUESTED** [review]

File: express-mongodb/signup-app.js (hunk 1 above)
Comment (zeshengchen) [reply express-mongodb/signup-app.js:18]: Please use dotenv module.

File: express-mongodb/signup-app.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          
+    return res.redirect('dashboard.html');
+})
+  
+  
+app.get('/',function(req,res){
+res.set({
+    'Access-control-Allow-Origin': '*'
+    });
+return res.redirect('signup.html');
```
Comment (zeshengchen) [inline express-mongodb/signup-app.js:63]: fix all indentation issue in the file

**Review by zeshengchen — CHANGES_REQUESTED** [review]

File: express-mongodb/package.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "body-parser": "^1.20.1",
```
Comment (zeshengchen) [inline express-mongodb/package.json:3]: As we discussed in the class, it prefers to use "-E" - extract version!

File: express-mongodb/signup-app.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+}).listen(3000)
```
Comment (zeshengchen) [inline express-mongodb/signup-app.js:64]: put PORT into .env file as well!

**Review by zeshengchen — CHANGES_REQUESTED** [review]

**Comment by sarinalalria** [comment]
Creating a new pull request to move folders, then I will fix the databse

**Pull request closed without merging**